src/zcbor_encode.c

```diff
@@ -547,6 +547,80 @@ bool zcbor_float32_put(zcbor_state_t *state, float input)
 }
 
 
+bool zcbor_float16_encode(zcbor_state_t *state, const float *input)
+{
+	return zcbor_float16_put(state, *input);
+}
+
+
+bool zcbor_float16_put(zcbor_state_t *state, float input)
```

Comment:
There is a lot of magic values used here, IMO it would look better if these were pulled out to `#define`s which had a comment associated with them.